_layouts/eip.html

```diff
@@ -41,4 +41,31 @@ <h1 class="page-heading">
 
   {{ content }}
 
+  <h2>Citation</h2>
+  <p>Please cite this document as:</p>
+  {% comment %}
+  IEEE specification for reference formatting:
+  https://ieee-dataport.org/sites/default/files/analysis/27/IEEE%20Citation%20Guidelines.pdf
+  {% endcomment %}
+  <p>{% include authorlist.html authors=page.author %}, "EIP-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}{% if page.status != "Final" %} [DRAFT]{% endif %}," <em>Ethereum Improvement Proposals</em>, no. {{ page.eip | xml_escape }}, {{ page.created | date: "%B %Y" }}. [Online serial]. Available: https://eips.ethereum.org/EIPS/eip-{{ page.eip | xml_escape }}.</p>
 </div>
+
+{% comment %}
+Article schema specification:
+https://schema.org/Article
+{% endcomment %}
+<script type="application/ld+json">
```

Comment:
I think actually this PR should be split into two PRs: the citation addendum above and this JSON schema. They serve two different purposes.

There was some discussion about the JSON schema arguing which is the best one, but I can't seem to find the discussion unfortunately.

Reply:
I still haven't found the discussion, just that I had a local branch adding `TechArticle`.

Also `_includes/head.html` already contains a jsonld section so it would be in conflict with this one.

Can you just pull the jsonld part out to a different conversation? Or perhaps pull out the citation into a new one, given we discussed jsonld here.

Reply:
This is in the current header:
```
  <script type="application/ld+json">
    {
      "@type": "WebSite",
      "url": "{{site.url}}",
      "name": "{{site.title}}",
      "description": "{{site.description}}",
      "@context": "https://schema.org"
    }
  </script>
```

This was my incomplete version:
```
  <script type="application/ld+json">
    {
      "@context": "https://schema.org",
      "@type": "TechArticle",
      "url": "{{site.url}}",
      "name": "{{page.title | jsonify}}",
      "description": "{{site.description | jsonify}}",
      "headline": "",
      "articleBody": "",
      "datePublished": "",
      "author": {
        "@type": "Person",
        "name": ""
      },
      "publisher": {
        "@type": "Organization",
        "name": "Ethereum Improvement Proposals (EIPs)",
        "logo": {
          "@type": "imageObject",
          "url": "http://example.com/images/logo.png"
        }
     },
     "image": {
       "@type": "imageObject",
       "url": "http://example.com/images/image.png"
     }
  </script>
```

Reply:
Removed JSON-LD at fc1ecca9749028a8f11b16c8857083a926bff97b

Reply:
don't see any controversy left here. good to merge?